rules/macos/credential_access_suspicious_web_browser_sensitive_file_access.toml

```diff
@@ -57,15 +57,15 @@ type = "eql"
 
 query = '''
 file where event.action == "open" and host.os.type == "macos" and process.executable != null and
- file.name : ("cookies.sqlite",
-              "key?.db",
-              "logins.json",
-              "Cookies",
-              "Cookies.binarycookies",
-              "Login Data") and
- ((process.code_signature.trusted == false or process.code_signature.exists == false) or process.name : "osascript") and
- not process.code_signature.signing_id : "org.mozilla.firefox" and
- not Effective_process.executable : "/Library/Elastic/Endpoint/elastic-endpoint.app/Contents/MacOS/elastic-endpoint"
+ file.name like~ ("cookies.sqlite",
+                  "key?.db",
+                  "logins.json",
+                  "Cookies",
+                  "Cookies.binarycookies",
+                  "Login Data") and
+ ((process.code_signature.trusted == false or process.code_signature.exists == false) or process.name == "osascript") and
+ not process.code_signature.signing_id == "org.mozilla.firefox" and
+ not Effective_process.executable like "/Library/Elastic/Endpoint/elastic-endpoint.app/Contents/MacOS/elastic-endpoint"
```

Comment:
Why not `!=` instead of like?

Reply:
Its more efficient in EQL to negate an exact match.